src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -9730,6 +9728,8 @@ DECLARE_API(GCWhere)
     return Status;
 }
 
+#ifndef FEATURE_PAL
+
 DECLARE_API(FindRoots)
 {
 #ifndef FEATURE_PAL
```

Comment:
Yep, there are a few nested #ifndef FEATURE_PAL here. this code could use some cleanup...